.github/workflows/build-and-push-cliain.yml

```diff
@@ -82,10 +71,12 @@ jobs:
                   (steps.get-ref-properties.outputs.branch-name-flattened == 'main' && 'latest' ||
                   steps.get-ref-properties.outputs.branch-name-flattened) ||
                 matrix.runtime == 'liminal' && steps.get-ref-properties.outputs.sha }}
-        uses: docker/build-push-action@v3.3.0
-        with:
-          context: ./bin/cliain
-          builder: ${{ steps.buildx.outputs.name }}
-          file: ./bin/cliain/Dockerfile
-          push: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'push' }}
-          tags: ${{ env.REGISTRY }}${{ env.IMAGE_NAME }}:${{ env.TAG }}
+        runs: |
```

Comment:
Typo: `runs` -> `run`
This workflow has not been run: https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/4681872533

Reply:
Fixed